### PR TITLE
add (expandedWidth, borderSize, collapsedWidth) parameters to the sidePanel component

### PR DESCRIPTION
**Pull request by Sofien-Sellami**
### Context
I've made these changes to give the option of changing the sidepanel parameters (expandedWidth, borderSize, collapsedWidth)

### Changes & Results
in this example I've expanded the width of the sidePanel 
           ``<SidePanel
                expandedWidth={500}
                side="right"
                activeTabIndex={rightPanelDefaultClosed ? null : 0}
                tabs={rightPanelComponents}
                servicesManager={servicesManager}
              />``

expanded-sidePanel

#### PR

- [x] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [x] My code has been well-documented (function documentation, inline comments,
  etc.)

#### Public Documentation Updates

- [x] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [x] OS: 
- [x] Node version: 
- [x] Browser:
  

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

**Comment by sedghi** [comment]
Sorry for my late response, but can you please update this based on the new side panel?

**Comment by sedghi** [comment]
git pull origin master --no-rebase
and solve the conflicts

File: platform/ui/src/components/SidePanel/SidePanel.tsx [hunk 1]
```diff
@@ -168,171 +182,171 @@ const SidePanel = ({ side, className, activeTabIndex: activeTabIndexProp, tabs,
...
-            className={classnames('text-primary-active', side === 'left' && 'rotate-180 transform')}
-          />
-        </div>
-        <div className={classnames('mt-3 flex flex-col space-y-3')}>
-          {_childComponents.map((childComponent, index) => (
-            <Tooltip
-              position={side === 'left' ? 'right' : 'left'}
-              key={index}
-              content={`${childComponent.label}`}
+        <>
```
Comment (jbocce) [inline platform/ui/src/components/SidePanel/SidePanel.tsx:185]: I see a lot of places like this where the indentation changed and I am not sure why. Could you please fix this?

**Review by jbocce — COMMENTED** [review]
Good change! Thanks!

See my one comment.

**Comment by jbocce** [comment]
Hmmm... the title of the PR says that also `borderSize` and `collapsedWidth` can be changed. I do not see those props. I am happy with just changing the `expandedWidth` for now if that is ok or are you planning on making more changes?

**Comment by Sofien-Sellami** [comment]
> Hmmm... the title of the PR says that also `borderSize` and `collapsedWidth` can be changed. I do not see those props. I am happy with just changing the `expandedWidth` for now if that is ok or are you planning on making more changes?

hi @jbocce, I fixed the stuff you asked for, and I don't think I'm going to add the bordersize and collapsedWidth props because I think they need to be fixed.

**Comment by jbocce** [comment]
> > Hmmm... the title of the PR says that also `borderSize` and `collapsedWidth` can be changed. I do not see those props. I am happy with just changing the `expandedWidth` for now if that is ok or are you planning on making more changes?
> 
> hi @jbocce, I fixed the stuff you asked for, and I don't think I'm going to add the bordersize and collapsedWidth props because I think they need to be fixed.

That makes sense and is my preference too.

**Review by jbocce — APPROVED** [review]
Approved. Thanks again!

**Comment by sedghi** [comment]
@Sofien-Sellami please `git pull origin master --no-rebase` and resolve conflicts and push again

**Comment by Sofien-Sellami** [comment]
> @Sofien-Sellami please `git pull origin master --no-rebase` and resolve conflicts and push again

HI @sedghi, there is no change

**Pull request closed without merging**